.github/main.go

```diff
@@ -177,25 +178,24 @@ func (ci *CI) withTestWorkflows(runner *dagger.Gha, name string) *CI {
 		WithJob(runner.Job("scan-engine", "engine scan")).
 		With(splitTests(runner, "testdev-", true, []testSplit{
 			{"cgroupsv2", []string{"TestProvision", "TestTelemetry"}, &dagger.GhaJobOpts{
-				// HACK: our main runners don't support cgroupsv2
-				Runner: []string{"ubuntu-latest"},
-				// NOTE: needed to silence redis warning logs in tests
-				SetupCommands: []string{"sudo sysctl -w vm.overcommit_memory=1"},
```

Comment:
We should keep this, we don't want the warning message in the redis telemetry test if possible.

Reply:
This does not work as expected in the new CI runner: https://github.com/dagger/dagger/actions/runs/13923575944/job/38962293256

<img width="1880" alt="image" src="https://github.com/user-attachments/assets/3d649c5b-6625-4432-9e31-e17a2c43225b" />

Would you like us to dig deeper into that, or are we OK keeping the assertion change?

Reply:
:thinking: that is a little odd. Why is `/proc` readonly?

If this is running inside a container, then it should be running as privileged. cc @cwlbraa, I know you dug into this before.

Not a blocker for merging, but still probably worth fixing.